apps/web/src/constants.ts

```diff
@@ -1,5 +1,5 @@
 export const artifacts = ['wasm', 'zkey']
 export const owner = 'privacy-scaling-explorations'
 export const repo = 'snark-artifacts'
-export const projects = ['poseidon', 'semaphore', 'sempahore-identity']
```

Comment:
I consistently making that typo.
using `projects` from `@zk-kit/artifacts` in the first place would have prevented this